docs/src/lib/functions.md

```diff
@@ -0,0 +1,86 @@
+```@meta
+CurrentModule = DataFrames
+```
+
+# Functions
```

Comment:
Isn't everything listed here a function?

Reply:
Yes, everything on that page is a function, I'm not sure this H1 header adds anything. We could remove it and only show the H2 headers of the function-groups (and bump them up to being H1 headers). But before I act on any of your other comments on where/how functions should be grouped, what are your thoughts on just listing all of the functions alphabetically in a single section? It might make navigation difficult since the functions wouldn't be grouped by concept, but the conceptual groupings are pretty arbitrary and no matter how we group them I'm sure we won't be able to make the groups intuitive to everyone. It will probably be easier to keep these docstrings up to date if we just list the functions in the same way they are exported in the `DataFrames.jl` source file.

Reply:
Actually, due to the presence of the isolated `meltdf`, I had misread this as a H2 heading. I'm OK with keeping it as H1, `meltdf` should simply be moved.

I'd say grouping is a better approach, but we don't need a lot of groups. Maybe one for basics, and one for grouping/joining & co.?